Elevato/Extand/ExtandEvents.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel;
+
```

Comment:
what is extand? I dont know what this means - make you class names meaningful

Reply:
its extend. meaning make larger. I think, i misspelled last time